src/dataflow/src/logging/differential.rs

```diff
@@ -26,9 +26,9 @@ use crate::activator::RcActivator;
 use crate::arrangement::manager::RowSpine;
 use crate::arrangement::KeysValsHandle;
 use crate::logging::ConsolidateBuffer;
-use crate::render::datum_vec::DatumVec;
 use crate::render::Permutation;
 use crate::replay::MzReplay;
+use repr::DatumVec;
 use repr::{Datum, Row, Timestamp};
```

Comment:
nit (that should have been automatically handled by tooling?) these two lines should be merged.

```suggestion
use repr::{Datum, DatumVec, Row, Timestamp};
```